geode-pulse/build.gradle

```diff
@@ -95,6 +94,9 @@ dependencies {
 
   // added only to ensure common version with other geode modules
   runtimeOnly('org.slf4j:slf4j-api')
+  runtimeOnly('org.apache.logging.log4j:log4j-slf4j-impl') {
+    exclude module: 'slf4j-api'
```

Comment:
Why are we including the slf4j above and excluding it below. If impl is runtime only then all its deps will also be runtime only. We should just collapse this into a single include.

Reply:
I believe by declaring `runtimeOnly('org.slf4j:slf4j-api')` explicitly will use the version we chose specifically, if we only do `runtimeOnly('org.apache.logging.log4j:log4j-slf4j-impl')`, then the version will depend on whatever version log4j-slf4j-impl uses.

doing this: 
runtimeOnly('org.slf4j:slf4j-api')
runtimeOnly('org.apache.logging.log4j:log4j-slf4j-impl')

will end up using the versions we declare for both jars.

If I understand how gradle dependency works.

Reply:
I'll declare both dependencies explicitly and remove the exclusion.

Reply:
Our BOM binds the version regardless of how it is imported. You can verify versions with ‘dependencies’ target. https://docs.gradle.org/current/userguide/viewing_debugging_dependencies.html

Reply:
With or without the explicit dependency on slf4j-api, the scan shows that we get version 2.13.1. So I'll leave it out.